packages/ckeditor5-fullscreen/src/handlers/classiceditor.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/**
+ * @module fullscreen/handlers/classiceditorhandler
+ */
+
+import { MenuBarView } from 'ckeditor5/src/ui.js';
+import type { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
+
+import AbstractEditorHandler from './abstracteditor.js';
+
+/**
+ * The classic editor fullscreen mode handler.
+ */
+export default class ClassicEditorHandler extends AbstractEditorHandler {
+	/**
+	 * An editor instance.
+	 */
+	protected override readonly _editor: ClassicEditor;
+
+	/**
+	 * @inheritDoc
+	 */
+	constructor( editor: ClassicEditor ) {
+		super();
+
+		this._editor = editor;
+
+		this._editor.on( 'destroy', () => {
+			this.disable();
+		} );
+	}
+
+	/**
+	 * Moves the editor UI elements to the fullscreen mode.
+	 */
+	public override enable(): void {
+		const editorUI = this._editor.ui;
+		const editorUIView = editorUI.view;
+
+		this.moveToFullscreen( editorUI.getEditableElement()!, 'editor' );
+		this.moveToFullscreen( editorUIView.toolbar.element!, 'toolbar' );
+
+		// In classic editor, the `dir` attribute is set on the top-level container and it affects the styling
+		// in both menu bar and toolbar (adding the side padding to the elements).
+		// Since we don't move the whole container but only parts, we need to reapply the attribute value manually.
+		// Decupled editor doesn't have this issue because there is no top-level container, so `dir` is set on each component separately.
+		this.getContainer().setAttribute( 'dir', editorUIView.element!.getAttribute( 'dir' )! );
+
+		if ( !this._editor.config.get( 'fullscreen.menuBar.isVisible' ) ) {
```

Comment:
Not a fan of this kind of early return. If we continue adding things into this method it will be easy to make a mistake and add it at the end without realizing that there's such `if` before.